core/ir/disassemble_shared.c

```diff
@@ -966,7 +966,8 @@ instr_disassemble_opnds_noimplicit(char *buf, size_t bufsz, size_t *sofar INOUT,
         }
     }
     if (is_evex_mask_pending) {
-        opnd = instr_get_src(instr, 0);
+        i = 0;
```

Comment:
Huh, I wonder if this code used to be inside the loop and was pulled out or sthg.  Maybe it would be nicer to have a new var with a descriptive name like `int mask_index = 0` instead of using a loop var when there's no loop?

Reply:
Yeah I think it got extracted from the loop in eba04c43e.

Would you prefer replacing i with 0 entirely or a `const mask_index = 0`?

Reply:
Does it need to be a writable var for when it's passed to `opnd_disassemble_noimplicit`?  If not, the `const mask_index` sounds nice.

Reply:
Ah, hmm. `opnd_disassemble_noimplicit` *declares* it as an in/out parameter but I don't see anything that actually writes to it in any architecture.